SessionUtilitiesKit/Database/Models/Setting.swift

```diff
@@ -5,252 +5,7 @@ import GRDB
 
 // MARK: - Setting
 
-public struct Setting: Codable, Identifiable, FetchableRecord, PersistableRecord, TableRecord, ColumnExpressible {
-    public static var databaseTableName: String { "setting" }
-    
-    public typealias Columns = CodingKeys
-    public enum CodingKeys: String, CodingKey, ColumnExpression {
-        case key
-        case value
-    }
-    
-    public var id: String { key }
-    public var rawValue: Data { value }
-    
-    let key: String
-    let value: Data
-}
+public typealias Setting = KeyValueStore
+public typealias EnumIntSetting = EnumInt
+public typealias EnumStringSetting = EnumString
```

Comment:
Can you please also add a `TODO` here to refactor the usage of these and remove them?